lib/capistrano/tasks/assets.rake

```diff
@@ -130,5 +130,6 @@ namespace :load do
   task :defaults do
     set :assets_roles, fetch(:assets_roles, [:web])
     set :assets_prefix, fetch(:assets_prefix, 'assets')
+    set :assets_root, fetch(:assets_root, release_path)
```

Comment:
Hmm, I think this breaks deployments. You are setting the default `:assets_root` based on  `release_path`, but the release path is still `nil` at this point. It isn't determined until later in the deploy flow.